lib/hashicorptools/code_deploy.rb

```diff
@@ -7,16 +7,59 @@
 require 'thor'
 
 module Hashicorptools
+  class RegionDeployment
+    attr_accessor :aws_region, :environment
+
+    def initialize(aws_region:, environment:)
+      @aws_region = aws_region
+      @environment = environment
+    end
+
+    def create_deployment(commit_id, commit_message)
+      Dotenv.load
```

Comment:
Why do we need to load `.env` here? Aren't all required credentials loaded from env vars set by `aws-vault`?